index.html

```diff
@@ -249,6 +244,26 @@ <h3>CONTACT US</h3>
       <p>auduomohkehinde@gmail.com</p>
     </footer>
 
+    <section id="card" class="card blur-bg">
+      <!-- <div>
+        <h2>Multi Post Story</h2>
+      <div>&times;</div>
+      </div>
+      <img src="img/spike.PNG" class="card-img" alt="">
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusamus, illum explicabo nemo eaque labore 
+        tempore doloremque, suscipit voluptatum officiis maxime dolorem earum. Aperiam, quos ex tenetur consequuntur 
+        nam temporibus fugiat.</p>
+        <div class="toolx">
+          <ul>
+            <li class="line">CSS</li>
+            <li class="line">html</li>
+            <li class="line">Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="project-link btnsee">See Project</button>
+        <button class="project-link btnsee">See Project</button> -->
```

Comment:
- [ ] [OPTIONAL] Kindly remove any commented code. to increase the code readability.

Reply:
This is not nice. The X button is working. This was my last review and you have successfully made me repeat the week.

Reply:
Check the project very well next time